scripts/filters/post-body-end/creative-commons.js

```diff
@@ -0,0 +1,14 @@
+/* global hexo */
+
+'use strict';
+
+const priority = hexo.config.inject_priority_creative_commons || 200;
```

Comment:
How about using
`hexo.config.inject_priority.creative_commons`
instead of
`hexo.config.inject_priority_creative_commons`

and `hexo.config.inject_priority.reward`, `hexo.config.inject_priority.wechat_subscriber`

Reply:
I have also considered this, but I can't determine the order of execution of the script, so in order to ensure that there is no error, need `(hexo.config.inject_priority||{}).xxxxxx`

Which do you think is better (I feel almost same) ?

1. `hexo.config.inject_priority_xxxx`
```yml
inject_priority_xxxx1: xx
inject_priority_xxxx2: xx
```

2. `(hexo.config.inject_priority||{}).xxxx`
```yml
inject_priority:
  xxxx1: xx
  xxxx2: xx
```

Reply:
除此之外，如果是外部的插件，需要优先级，我更倾向于与插件的配置放在一起，在`hexo`的配置文件中配置
In addition, if it's an external plug-in that needs priority, I prefer to put it with the configuration of the plug-in, in the configuration file of 'hexo'

like this [hexo-cake-moon-menu](https://github.com/jiangtj-lab/hexo-cake-moon-menu)
If need priority, have to add this in hexo `_config.yml`.
```yml
moon_menu:
  priority: xx
  back2top:
    enable: true
  back2bottom:
    enable: true
```

Reply:
确实，短路表达式可能会出现报错。插件一般用lodash merge或者Object.assign来应用默认配置：
https://github.com/hexojs/hexo-generator-index/blob/08fa5fafef2e364a2147e5000a87ee77bc585fd5/index.js#L5-L8

https://github.com/wzpan/hexo-generator-search/blob/2482d2c24d5ecb268be567390a309b99ba0a4665/index.js#L1-L7

```
hexo.config.inject_priority = Object.assign({
  creative_commons: 200,
  xxxx: 100,
  xxxx: 0
}, hexo.config.inject_priority);
```

Reply:
> hexo.config.inject_priority = Object.assign({
>   creative_commons: 200,
>   xxxx: 100,
>   xxxx: 0
> }, hexo.config.inject_priority);

做不到啊，这个配置不在过滤器内部而在外边的脚本中，脚本载入非常早，而且hexo控制的，我们不知道脚本载入的先后顺序呀 😂

Reply:
或者保持原状，仅重构`post-copyright`，类似于 https://github.com/iissnan/hexo-theme-next/pull/1667 ，在`_config.yml`中给几个选项，让用户决定在文末展示哪些版权信息。
大量使用`inject_priority`会使得NexT主题向一个奇怪的方向发展…

Reply:
有点赞同保持原状😂

奇怪的方向。。。我比较担心会出现许多js脚本。

看样子我还是太着急了，等github的包管理出了之后，再搞。。。把一些不重要的功能移出去，比如wechat_subscriber

Reply:
关于重构post-copyright，我更倾向于交给其他人维护，由别人提供插件，我们只需要告诉用户，这里有xxx插件可以用就好了，甚至可能有多款可以选 :joy:

但现在只有我们写给插件。。。

Reply:
是的，theme-inject更加适合第三方插件使用，而主题内能够用模版做的事情还是尽量用模版去做...